tools/stress-cluster/services/Stress.Watcher/src/PodEventHandler.cs

```diff
@@ -78,7 +77,9 @@ public async Task Watch(CancellationToken cancellationToken)
             {
                 try
                 {
-                    var listTask = Client.ListPodForAllNamespacesWithHttpMessagesAsync(
+                    Logger.Information("Starting pod watch");
+
+                    var listTask = Client.CoreV1.ListPodForAllNamespacesWithHttpMessagesAsync(
```

Comment:
I read a little bit about the CoreV1 and BatchV1 for pod and job, still couldn't figure out the difference between this, and the previous implementation

Reply:
other than those 2, lgtm

Reply:
@ckairen My understanding is they moved the method organization into classes within their respective API groups, vs. just having them all at the top level like before. A lot of this code is auto-generated from the kubernetes REST spec anyway.

[pod spec, corev1 api group](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/), [job spec, batchv1 api group](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/job-v1/).

Reply:
ahh ok, so it's their API change